server_addon/houdini/client/ayon_houdini/plugins/publish/validate_usd_render_product_paths.py

```diff
@@ -26,6 +26,7 @@ class ValidateUSDRenderProductPaths(pyblish.api.InstancePlugin):
     hosts = ["houdini"]
     label = "Validate USD Render Product Paths"
     optional = True
+    enabled = False
```

Comment:
Does this just hint that this should be configurable in studio settings and disabled by default? Why are you disabling it?

Reply:
> Does this just hint that this should be configurable in studio settings and disabled by default?

I think yes. tbh, I have no idea what is this `$HIP/renders/$HIPNAME:r/$OS/$HIPNAME:r.$OS.$F4.exr` and it was not in the node on creation by default. and that note encouraged me to just disable the validator 😅
```
(colorbleed): This workflow might be relatively Colorbleed-specific
```

Reply:
Yup, so basically what happens is that the USD Render ROP writes out an intermediate `__render__.usd` file or alike - however, by default that's just a static path, not a render file for your particular workfile if I recall correctly. And thus, if the user quickly submits to deadline three files from the same work folder it basically means that the each job would overwrite that `render.usd` file the other two jobs are also writing - hence you can't submit multiples at once. By making sure those files have unique names (e.g. containing `$HIPNAME` that solves that particular issue).